src/resources/Constants.java

```diff
@@ -0,0 +1,9 @@
+//user defined package
+package resources;
```

Comment:
specified package doesn't show a relationship with other packages of the same program modify to something like 
```suggestion
package com.debonair.constants;
```